### PR TITLE
Change WordPress blue to update value in WP 5.3

### LINKED ISSUE #3475: Update blue to use WordPress's new blue
## Bug Description

Update buttons and highlights to use updated WordPress colours. 

## Expected Behaviour

All buttons and icons should use WordPress blue. 

## Screenshots
WordPress's new blue
Screenshot from 2019-10-09 15-18-56

Current blue
Screenshot from 2019-10-09 15-14-31
Screenshot from 2019-10-09 15-15-08

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary
Fixes #3475 

## Checklist

- [x] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -2,18 +2,18 @@ module.exports = {
 	plugins: [
 		require( '@wordpress/postcss-themes' )( {
 			defaults: {
-				primary: '#0085ba',
+				primary: '#007cba',
```
Comment (miina) [inline postcss.config.js]: There are some hardcoded values in the CSS currently using `#007cba`, is this a coincidence or would it be better to switch those to using the theme values? Thoughts?

Comment (swissspidy) [reply postcss.config.js]: Good catch! We should use the theme values whenever possible to avoid regressions and inconsistencies.

Comment (spacedmonkey) [reply postcss.config.js]: A quick search of the code base shows on these references to these colours. https://github.com/ampproject/amp-wp/search?q=0085ba&unscoped_q=0085ba

Where is the issue here?

Comment (spacedmonkey) [reply postcss.config.js]: Fixed in https://github.com/ampproject/amp-wp/pull/3494/commits/adf9cbe29c27f6dab15e4e54d64d152213637047

**Pull request merged**